docs/cloud/features/scheduler/scheduler.md

```diff
@@ -199,14 +219,9 @@ SQLMESH__GATEWAYS__GATEWAY_B__CONNECTION__TYPE=<connection type>
 # <Gateway B connection settings>  
 ```
 
-Once you have set up both sets of environment variables in a file named `local.env`, you can run the following command to start the executor:
-
-```shell
-docker run -d --env-file local.env tobikodata/tcloud:latest -- executor run
-docker run -d --env-file local.env tobikodata/tcloud:latest -- executor apply
-```
+For more details, including secure secret management options, refer to the [Helm chart](./scheduler/hybrid-executors/README.md) or [Docker Compose](./scheduler/hybrid-executors-docker-compose.md) documentation.
 
-After the executors are properly configured, they will appear in the cloud UI where they can be used to apply plans and execute scheduled runs.
+After the executors are properly configured and running, they will appear in the cloud UI where they can be used to apply plans and execute scheduled runs.
```

Comment:
Can you add a conceptual point in the ingress section on how this can be safely deployed in a private vpc and reach out to specific IPs to send metadata to Tobiko Cloud? Can you list out those IPs assuming they're static?

Reply:
will this require customizing the helm chart beyond the values.yaml template?

Reply:
There is no ingress related to this deployment and nothing that the user will need to change if deploying to a private VPC. They would just need to allow-list the IPs in their network configuration but this is outside of the scope of the executor deployment and likely a different team would be involved in that (security/networking team within the company).

Reply:
I thought about this. Can you put something in writing on how the helm chart/docker compose will securely work in a private VPC and reach out to static IPs? You can list out fake placeholders.

This will be a very common question we should have easy docs for. 

Example: 
- Whitelist these IPs
- Here's how this will look with placeholders in this `values.yaml` or `compose.yaml`
- The hybrid runners will only reach out to Tobiko Cloud to send data

Reply:
Also added "Network Configuration" section saying what needs to be done if behind a private network and listed the IPs.